app/graphql/types/profile.rb

```diff
@@ -87,8 +92,17 @@ def last_scanned(args = {})
       end
     end
 
+    def hosts
+      ::CollectionLoader.for(policy_or_report.class, :hosts)
+                        .load(policy_or_report)
+    end
+
     private
 
+    def policy_or_report
+      object.policy_object || object
```

Comment:
Observation: I'm not sure if this would fail if the `policy_object` is found, as it is not type of a `Profile`.

Reply:
It's only used for internal lookups.